### DISCUSSION TITLE
yolov5/tutorials/clearml_logging_integration/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#14704] [Docs]
# yolov5/tutorials/clearml_logging_integration/

Learn how to use ClearML for tracking YOLOv5 experiments, data versioning, hyperparameter optimization, and remote execution with ease.

https://docs.ultralytics.com/yolov5/tutorials/clearml_logging_integration/

**Comment by JLZ22** [comment]
When you are giving instructions on how to change the project and task names, it looks like you are referring to a `train.py` script which has been written already. Where can I find this script?

**Reply by glenn-jocher** [reply]
@JLZ22 the `train.py` script you're referring to is part of the YOLOv5 repository. You can find it in the root directory of the YOLOv5 project. This script is used to train YOLO models and includes options for setting project and task names, among other parameters. For detailed instructions on how to use it with ClearML, you can refer to the ClearML integration guide provided in the Ultralytics documentation. If you have any further questions, feel free to ask!
